### DISCUSSION TITLE
Version 8.3 Stable

### DISCUSSION THREAD

**Discussion by Arnova8G2** [Arnova8G2/Gcam_8.7.250.44#57] [Ideas]
**Change logs :**

- Fixed bugs from 8.2 version
- Fixed all video modes for many devices
- Add createCustomCaptureSession force some devices with operations mode
- Enabled gboard fix for dev by default.

All modes seem to work very well how it should be.
**Very Stable**
 
 
**APK certificate fingerprints:** 
SHA-1: e0be67031a86c4178239256f80d631d94451e263 
  
APK file name: **arn.android.gcam**  
Gcam-8.7.250.build-8.3.230603.0103.apk 
 
APK file hashes: 
MD5: d2e668d049371105f071cd0b3205e4cf 
SHA-1: ad7b3b0415fdc9a47aaaefcc696aee4c186cbfae 

APK file name: **com.google.android.GoogleCamera**  
Gcam-8.7.250.build-8.3.230603.0105.apk 
 
APK file hashes: 
MD5: 456b17ed14c830d94c4e5abffbda6ac4 
SHA-1: e6776ef2bbcbb8200408bb80f9b835ff934da69e 

APK file name: **com.samsung.android.scan3d**  
Gcam-8.7.250.build-8.3.230603.0108.apk 
 
APK file hashes: 
MD5: 426b5aee874b427d34f4961e3f118866 
SHA-1: cbe93e4f425954afc6aed26cbd06c6843fff3d71 

APK file name: **com.ss.android.ugc.aweme**  
Gcam-8.7.250.build-8.3.230603.0112.apk 
 
APK file hashes: 
MD5: 7ca22bf25fc1deb9b8ce5d3a9815ae85 
SHA-1: 8443e28611d417c8d3de550927e0a85f611cf741 

APK file name: **org.codeaurora.snapcam**  
Gcam-8.7.250.build-8.3.230603.0118.apk 
 
APK file hashes: 
MD5: 9098bcabba3db844d13b8ed678cd856e 
SHA-1: 791571a78f90b67339af7155c37b714afcf06b49 

<hr /><em>This discussion was created from the release <a href='https://github.com/Arnova8G2/Gcam_8.7.250.44/releases/tag/V8.3'>Version 8.3 Stable</a>.</em>

**Comment by wezzyd86** [comment]
Sounds daft but were meant to be using the Samsung one right?

**Comment by AnonymousWP** [comment]
Would this work well OOTB with the OnePlus 7 Pro for example? Or are config files (from other developers) still necessary?

**Reply by Arnova8G2** [reply]
Works great on OP7Pro, I'm installing PixelExperience to see if works well.
in dev search for "memory" and replace 1400 with -1 this will make memory usage dynamic

**Reply by Arnova8G2** [reply]
For moment only bug with slow motion
enable stream config to normal and in dev use 240 ( camcorder.capture_rate )

**Reply by Arnova8G2** [reply]
FPS is restricted for slow motion

Normal 
![Screenshot_20230603-133655_MediaInfo~2](https://github.com/Arnova8G2/Gcam_8.7.250.44/assets/40323300/d58f02b4-41f0-47b2-8309-8ef6cdf7f7aa)

Forced
![Screenshot_20230603-133645_MediaInfo](https://github.com/Arnova8G2/Gcam_8.7.250.44/assets/40323300/528cae2c-80b6-46d4-af22-411feae673da)
![Screenshot_20230603-134313_Appareil photo](https://github.com/Arnova8G2/Gcam_8.7.250.44/assets/40323300/4f4de0f2-b45d-46bf-a6e4-09c8114f1850)

**Reply by AnonymousWP** [reply]
Great, thanks for the reply. Would it also work well with any 7 Pro config? For example Greatness' config.
